cdx-cli/jit-db-setup/jit-workload-account/7.update-task-definition.sh

```diff
@@ -83,8 +95,15 @@ aws secretsmanager update-secret \
     --secret-id $SECRET_NAME \
     --secret-string "$NEW_SECRETS"
 
-# Create updated task definition
-log "Creating updated task definition..."
+log "Creating CloudWatch Log Group..."
+aws logs create-log-group --log-group-name $LOG_GROUP_NAME_3
+
+aws logs tag-log-group \
+    --log-group-name $LOG_GROUP_NAME_3 \
+    --tags '{"Purpose": "database-iam-jit", "created_by": "cloudanix"}'
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code/findings/d803ac66c50fdc16/PR/2/artifacts/Divyansh/latest/secrets/show" target='_blank'>Click here for more details</a></b>